gcovr/gcov.py

```diff
@@ -635,6 +674,7 @@ def run_gcov_and_process_files(
     # it probably includes extra arguments.
     cmd = options.gcov_cmd.split(' ') + [
         abs_filename,
+#        "--demangled-names", #only modern versions of gcov support name demandling
```

Comment:
This option was added in 2013 https://github.com/gcc-mirror/gcc/commit/fafee80bbba6dcfdfc587d582986f7a77e8cdd03
So i think it is part of gcc since version 4.9

I think it would be good to have the option to use the demangler from gcov.